docs/content/preview/yugabyte-platform/manage-deployments/xcluster-replication/bidirectional-replication.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Bidirectional replication using xCluster
+headerTitle: Bidirectional replication
+linkTitle: Bidirectional replication
+description: Bidirectional replication using xCluster
+headContent: Replicate data in both directions
+menu:
+  preview_yugabyte-platform:
+    parent: xcluster-replication
+    identifier: bidirectional-replication
+    weight: 50
+type: docs
+---
+
+With bidirectional replication, both universes can perform reads and writes, and writes are replicated in both directions. This is also referred to as [Active-active multi-master](../../../architecture/docdb-replication/async-replication/#active-active-multi-master).
+
+You set up bidirectional replication using YugabyteDB Anywhere by creating two separate non-transactional xCluster replication configurations. Under this scenario, you create a replication configuration from universe A to universe B, and then you create a new replication configuration from universe B to universe A.
+
+YugabyteDB Anywhere detects an xCluster setup is for bidirectional if the tables that are being added to the replication are already part of a replication in the reverse direction (from A to B). In this scenario, YugabyteDB Anywhere skips the full copy for tables on universe B that are already in replication.
+
+{{<note title="Bidirectional xCluster deployment">}}
+Due to its operational complexity, bidirectional xCluster is not recommended for most use cases. If you are considering a bidirectional xCluster deployment, contact {{% support-general %}}.
+{{</note>}}
+
+## Limitations
+
+- Bidirectional replication can only be done using non-transactional replication. Transactional replication is not supported because transactional replication puts the target universe in read-only mode. For more information, see [Asynchronous replication modes](../../../../architecture/docdb-replication/async-replication/#asynchronous-replication-modes).
```

Comment:
Read only mode is not the reason for it.
It's not supposed since xcluster is async with last writer wins semantics. That is not transactional or ACID compliant.

Reply:
@shahrooz1997

Reply:
I agree that mentioning read-only convey more useful information to the user rather than explaining why it was decided to go with this approach.